packages/ses/src/commons.js

```diff
@@ -133,6 +133,8 @@ export const { prototype: weakmapPrototype } = WeakMap;
 export const { prototype: weaksetPrototype } = WeakSet;
 export const { prototype: functionPrototype } = Function;
 export const { prototype: promisePrototype } = Promise;
+// eslint-disable-next-line no-empty-function
+export const { prototype: iteratorPrototype } = getPrototypeOf(function*(){});
```

Comment:
I'd appreciate if someone could verify whether this is the iterator prototype or generator prototype. I managed to confuse myself while implementing this and while I think this is correct, there's doubt.

cc @mhofman